frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_page.dart

```diff
@@ -375,9 +375,14 @@ class _UnscheduledEventsButtonState extends State<UnscheduledEventsButton> {
                   _popoverController.show();
                 }
               },
-              child: FlowyText.regular(
-                "${LocaleKeys.calendar_settings_noDateTitle.tr()} (${state.unscheduleEvents.length})",
-                fontSize: 10,
+              child: Tooltip(
+                message: state.unscheduleEvents.isEmpty
+                    ? "Unscheduled events will show up here"
+                    : "${state.unscheduleEvents.length} events are unscheduled",
```

Comment:
Localize this message with a placeholder

Reply:
Can you provide me some examples, to understand this more clearly.

Reply:
Am guessing here you mean , storing the string in the common .json file and creating a LocaleKeys to get it @MayurSMahajan

Reply:
- Add the translations to AppFlowy/frontend/resources/translations/en.json.
- Execute the command sh scripts/code_generation/language_files/generate_language_files.sh in the frontend directory.
- Use something similar to LocaleKeys.document_imageBlock_upload_placeholder.tr() to replace the hardcoded text.